shared/locales/media/media.en-US.properties

```diff
@@ -0,0 +1,14 @@
+# Media
+
+byteUnit-B  = Bytes
+byteUnit-KB = KB
+byteUnit-MB = MB
+byteUnit-GB = GB
+byteUnit-TB = TB
+byteUnit-PB = PB
+byteUnit-EB = EB
+byteUnit-ZB = ZB
+byteUnit-YB = YB
+
+videoDescriptionShort = Video created at %I:%M %p, %B %e, %Y
+imageDescriptionShort = Image created at %I:%M %p, %B %e, %Y
```

Comment:
The word "created" is problematic. If the image or video was not captured by the camera, then we don't know when it was created. The date could just as well be when it was downloaded or when it was received (by sms or email, for example). To keep it simple you could just replace "created at" with "dated".  If you want to be more complicated, you could check whether the photo or video was stored under DCIM/, in which case you could use "Photo taken at" or "Video recorded at".